geom/src/polygon.rs

```diff
@@ -161,24 +161,14 @@ impl Polygon {
             &self.points
         }
     }
-    pub fn into_points(mut self) -> Vec<Pt2D> {
```

Comment:
Polygon -> points doesn't make sense generally. Callers should ask for the outer ring or the holes or all of them, and do whatever's appropriate